stackit/internal/services/argus/scrapeconfig/resource.go

```diff
@@ -552,7 +566,9 @@ func toCreatePayload(ctx context.Context, model *Model) (*argus.CreateScrapeConf
 		MetricsPath:    conversion.StringValueToPointer(model.MetricsPath),
 		ScrapeInterval: conversion.StringValueToPointer(model.ScrapeInterval),
 		ScrapeTimeout:  conversion.StringValueToPointer(model.ScrapeTimeout),
-		Scheme:         conversion.StringValueToPointer(model.Scheme),
+		// potentially lossy conversion, depending on the allowed range for sample_limit
+		SampleLimit: utils.Ptr(float64(model.SampleLimit.ValueInt64())),
```

Comment:
This field is declared as a float in the API's specifications, hence why this conversion is necessary.

This isn't blocking; when the specifications are updated at a later point this conversion will be removed